src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/IsExternalInit.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>
+    /// Reserved to be used by the compiler for tracking metadata.
+    /// This class should not be used by developers in source code.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public static class IsExternalInit
```

Comment:
I just want to clarify... the issue lists the type as `sealed`:
https://github.com/dotnet/runtime/issues/34978#issuecomment-614845405
but here it's `static`.  `static` rather than `sealed` was actually intended?

Reply:
Yes, after closer examination of other types used by the compiler in modreqs, we decided this should be a static class similar to `IsVolatile`